pages/api/stacking-promotions/redeemStackable.ts

```diff
@@ -0,0 +1,55 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { validateRequestedCart } from "../../../utils/validateRequestedCart";
+import { calculateCartTotalAmount } from "../../../utils/calculateCartTotalAmount";
+import { removeDuplicatedPromoObjects } from "../../../utils/removeDuplicatePromo";
+import { PromotionStackableObj } from "../../types";
+import { client } from "../voucherifyClient";
+
+const customer = {
+  object: "customer",
+  source_id: "test_customer_id_1",
```

Comment:
is `test_customer_id_1` some special const?

Reply:
this source_id is created on all accounts 
by default